File: Java/01 - Introduction to Programming/Task03/menu.java
import java.util.*;

public class menu
{
  public static void main(String[] args)
  {
    Scanner keyboardInput = new Scanner(System.in);
    System.out.println("Please enter your 3 favourite foods");
    System.out.print("Item 1: ");
    String item1 = keyboardInput.nextLine();
    System.out.print("Item 2: ");
    String item2 = keyboardInput.nextLine();
    System.out.print("Item 3: ");
    String item3 = keyboardInput.nextLine();

    System.out.println("Order confirmation! You have ordered:");
    System.out.println(item1);
    System.out.println(item2);
    System.out.println(item3);
  }
}

File: C++/01 - Introduction to Programming/Task03/menu.cpp
#include <iostream>

using namespace std;

int main()
{
    string item1;
    string item2;
    string item3;
    
    cout<<"Please enter your 3 favourite foods"<<endl;
    cout<<"Item 1: ";
    cin>>item1;
    cout<<"Item 2: ";
    cin>>item2;
    cout<<"Item 3: ";
    cin>>item3;

    cout<<"Order confirmation! You have ordered:"<<endl;
    cout<<item1<<endl;
    cout<<item2<<endl;
    cout<<item3<<endl;

    return 0;
}